netbeans.apache.org/src/content/community/index.asciidoc

```diff
@@ -31,8 +31,8 @@ The Apache NetBeans Community link:mailing-lists.html[communicates in many diffe
 == link:/participate/index.html[Participate]
 We welcome link:/participate/index.html[all kind of contributions]. And there are many ways to contribute: see how you can participate in Apache NetBeans.
 
-== link:events.html[NetBeans Events]
-NetBeans users and developers participate in link:events.html[different events]. See how this works.
+== link:/events/index.html[NetBeans Events]
+NetBeans users and developers participate in link:/events/index.html[different events]. See how this works.
```

Comment:
Regular expression is `^https?://.*apache.org/events/current-event`. Should we use the absolute URL(http(s)://netbeans.apache.org/events/current-event/index.html)? (see: http://trafficcontrol.apache.org/events/current-event/)

Reply:
I don't think we have anything like a "current-event/" in NetBeans. So I added a redirection from "/events/current-event/" to "/events/index.html" in the .htaccess. We can always undo this later on, in case we have something like a "current-event".

Reply:
I think the point of this is to redirect to https://www.apache.org/events/current-event ?  And note, doesn't seem to work with a trailing slash.

Internal links should always be without the domain anyway, but this one isn't.

Reply:
Actually, not redirect, just a link?  https://www.apache.org/events/README.txt

Reply:
Ah! So it's not that we have to create an "/events/current-event/" folder! It means that we have to redirect to https://www.apache.org/events/current-event ! I'll learn about that and see how to best integrate this into our pages.

Reply:
Hi @neilcsmith-net I've asked about this by email and it seems we're requested to add an image as well. Details on the dev mailing list.